server/routers/authenticate/authProviders/passkeys.ts

```diff
@@ -0,0 +1,714 @@
+// src/server/routers/passkeys.ts
```

Comment:
All these procedures should be using Prisma transactions to make sure we don't leave orphan/incomplete records behind if an error is thrown at some point (e.g. if an error happens before we the query to delete the used challenges run).